app.json

```diff
@@ -55,7 +56,24 @@
     "supportsTablet": true,
     "infoPlist": {
       "ITSAppUsesNonExemptEncryption": false,
-      "NSCameraUsageDescription": "This app uses the camera to scan QR codes for addresses."
+      "NSCameraUsageDescription": "This app uses the camera to scan QR codes for addresses.",
+      "LSApplicationQueriesSchemes": [
+        "metamask",
+        "trust",
+        "zerion",
+        "rainbow",
+        "ledgerlive",
+        "cbwallet",
+        "phantom",
+        "argent",
+        "imtoken",
+        "spot",
+        "omni",
+        "onto",
+        "safe",
+        "tokenpocket",
+        "exodus"
```

Comment:
Don't we have a source we could use to generate those? I guess this list is way bigger?